packages/components/Button/src/Button.tsx

```diff
@@ -20,12 +26,16 @@ export const Button = compose<IButtonType>({
       content,
       onAccessibilityTap = userProps.onClick,
       accessibilityLabel = userProps.content,
+      componentRef = userProps.componentRef != undefined ? userProps.componentRef : React.useRef(null),
```

Comment:
I believe this can be written as: 

`componentRef = userProps.componentRef ?? React.useRef(null)`

See [Nullish Operator](https://www.javascripttutorial.net/es-next/javascript-nullish-coalescing-operator/#:~:text=ES2020%20introduced%20the%20nullish%20coalescing,double%20question%20marks%20(%20%3F%3F%20).&text=It%20returns%20the%20right%20operand,is%20either%20null%20or%20undefined%20.)

Reply:
This is same for all the usages below

Reply:
Oh nice! Will definitely use that.